import-service/serverless.ts

```diff
@@ -0,0 +1,96 @@
+const serverlessConfiguration = {
+    service: 'import-service',
+    provider: {
+      name: 'aws',
+      runtime: 'nodejs20.x',
+      stage: 'dev',
+      region: 'eu-west-1',
+      environment: {
+        PRODUCTS_TABLE: 'Products',
+        STOCK_TABLE: 'Stock',
+        REGION: 'eu-west-1',
+        BUCKET: 'import-service-csv',
+      },
+      iamRoleStatements: [
+        {
+          Effect: 'Allow',
+          Action: ['sqs:ReceiveMessage', 'sqs:SendMessage'],
+          Resource: "*"
```

Comment:
Ideally to make more granular permissions for security reasons - in this case you need only SendMessage and for specific queue, not *

Reply:
done